src/2025h2/field-projections.md

```diff
@@ -0,0 +1,98 @@
+# Design a language feature to solve Field Projections
+
+| Metadata         |                                                                                  |
+|:-----------------|----------------------------------------------------------------------------------|
+| Point of contact | @BennoLossin                                                                     |
+| Teams            | <!-- TEAMS WITH ASKS -->                                                         |
+| Task owners      | <!-- TASK OWNERS -->                                                             |
+| Status           | Proposed                                                                         |
+| Tracking issue   |                                                                                  |
+| Zulip channel    | N/A                                                                              |
+
+## Summary
+
+Figure out the best design for field projections. Update the existing [Field Projections RFC] or
+author a new one and implement it for use in nightly via a lang experiment.
+
+[Field Projections RFC]: https://github.com/rust-lang/rfcs/pull/3735
```

Comment:
It seems like the thinking has progressed since this RFC, in which case I encourage you to also link the latest draft.